tweets/YYYY/MM/ama-announce-01.tweet

```diff
@@ -0,0 +1 @@
+Interested in helping, teaching, hosting a CodeRefinery workshop, contributing to a lesson material, using GitLab service, etc.? Join us in a community call "Ask Me Anything" on Aug 23, 12:00 CEST! Connection details will be announced later.
```

Comment:
```suggestion
Interested in helping, teaching, hosting a CodeRefinery workshop, contributing to lesson material, using GitLab service, etc.? Join us in a community call "Ask Me Anything" on Aug 23, 12:00 CEST! Connection details will be announced later.
```

Reply:
Not sure how to merge from mobile, but thanks for the sui!